rl_algorithms/common/buffer/wrapper.py

```diff
@@ -1,31 +1,33 @@
 # -*- coding: utf-8 -*-
-"""Prioritized Replay buffer for algorithms.
+"""Wrappers for buffer.
 
-- Author: Kyunghwan Kim
-- Contact: kh.kim@medipixel.io
+- Author: Kyunghwan Kim & Euijin Jeong
+- Contact: kh.kim@medipixel.io & euijin.jeong@medipixel.io
 - Paper: https://arxiv.org/pdf/1511.05952.pdf
          https://arxiv.org/pdf/1707.08817.pdf
 """
 
 import random
-from typing import Any, List, Tuple
+from typing import Any, Tuple
 
 import numpy as np
 import torch
 
-from rl_algorithms.common.buffer.replay_buffer import ReplayBuffer
+from rl_algorithms.common.abstract.buffer import BaseBuffer, BufferWrapper
 from rl_algorithms.common.buffer.segment_tree import MinSegmentTree, SumSegmentTree
 
 device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
 
 
-class PrioritizedReplayBuffer(ReplayBuffer):
-    """Create Prioritized Replay buffer.
+class PERWrapper(BufferWrapper):
+    """Prioritized Experience Replay wrapper for ReplayBuffer.
```

Comment:
`Prioritized Experience Replay wrapper for ReplayBuffer -> Prioritized Experience Replay wrapper for Buffer`

Reply:
resolved at 7348486.